src/datasource.ts

```diff
@@ -38,24 +39,28 @@ export class DataFrameDataSource extends DataSourceApi<DataframeQuery> {
   }
 
   async query(options: DataQueryRequest<DataframeQuery>): Promise<DataQueryResponse> {
-    const validTargets = options.targets.filter(isValidQuery);
+    try {
+      const validTargets = options.targets.filter(isValidQuery);
 
-    const data = await Promise.all(
-      validTargets.map(async (query) => {
-        const tableData = await this.getDecimatedTableData(query, options.range, options.maxDataPoints);
+      const data = await Promise.all(
```

Comment:
Not sure if just the diff tool but this spacing does seem off.

Reply:
I think it's just the diff tool showing the indent from wrapping this in a try-catch. The Grafana toolkit formatter uses 2 spaces.